src/AndcultureCode.CSharp.Sitefinity.Testing/Extensions/RestResponseResultExtensions.cs

```diff
@@ -0,0 +1,33 @@
+﻿using AndcultureCode.CSharp.Sitefinity.Core.Models.Services;
+using AndcultureCode.CSharp.Sitefinity.Testing.Constants;
+using Shouldly;
+
+namespace AndcultureCode.CSharp.Sitefinity.Testing.Extensions
+{
+    public static class RestResponseResultExtensions
+    {
+        public static void ShouldBeExpectedStatusCode<T>(this RestResponseResult<T> restResponseResult)
```

Comment:
Suggestion: Since this is a single line function, consider converting to an expression body.